packages/playback-core/src/hls.ts

```diff
@@ -0,0 +1,9 @@
+// @ts-ignore
+import HlsMin from 'hls.js/dist/hls.min.js';
+import type HlsClassType from 'hls.js';
```

Comment:
is it possible that we're including hls.js twice?

Also, since we're not no longer using the default import, there's no benefit of marking it as externals. Maybe that's another direction? https://github.com/muxinc/elements/blob/main/packages/playback-core/package.json#L41

Reply:
ah, just noticed that line changed to `import type`, so, we won't be including it twice, probably.

Reply:
`import type` should avoid double inclusion. Let me confirm the bundles wrt `HlsMin` though.

Reply:
ah you're specifically concerned about app level "double" bundles (more specifically, bundling both `hls.js/dist/hls.min.js` and `hls.js`). Yeah, for folks who have the dependency twice, that's definitely a possibility. Given that this is a stopgap solution, and given that your proposed change won't make the "double bundle" situation any better, I recommend leaving as is.